### ISSUE TITLE
README assumptions

### ISSUE THREAD

**Issue by erikdonohoo** [trylled/media-manager#6]
Hey @trylled, Erik from GoReact here. A few things I've noticed just trying to get this running.

It looks like you make the assumption I am using Visual Studio for your setup instructions. You can't take anything for granted in your instructions on how to get things running. You need to assume I have none of the tools and things you use on my machine.

I will go ahead and get that installed, but I would prefer you call out all the steps to get things running. Even better, use something like Docker or Vagrant to abstract all the setup away so I can just run one command to get going. Anyway, Im' going to see if I can get your project running, but you should update your README with more explicit instructions on how to get everything setup and running.

**Comment by erikdonohoo** [comment]
Yeah, so I have no idea what the NUGET package manager is or how to use it inside Visual Studio. I'm not super familiar with Visual Studio or C#, but thats fine, ultimately we just need your setup instructions updated with more specific insructions on how to get the database/backend setup especially. I got your front-end installed with npm no problem, but will need more from you to go on. Also, I noticed you didn't write any tests for this project. Do you intend to still do that @trylled ?

**Comment by trylled** [comment]
@erikdonohoo, Thanks for the feedback. I added a simple test using Jasmine, also updated README on how to run test.

For the database part, I included some images in the README on how to run update-database. 

After running "update-database" command it will automatically create a new database if it is not available and will run migration files.

**Comment by erikdonohoo** [comment]
@trylled Unfortunately this doesn't seem possible for me on a mac. I have the Mac version of Visual Studio installed, but there are limitations and I can't use your instructions to run your project. You will either need to submit a video to us of you walking through your project working on your machine, or get it hosted and send us a URL we can view to play with it ourselves.
